qcodes/instrument_drivers/oxford/MercuryiPS_VISA.py

```diff
@@ -355,6 +362,18 @@ def _ramp_safely(self) -> None:
                 while slave.ramp_status() == 'TO SET':
                     time.sleep(0.1)
 
+    def is_ramping(self) -> bool:
+        """
+        Returns True if any axis has a ramp status that is either 'TO SET' or
+        'TO ZERO'
+        """
+        ramping_statuus = ['TO SET', 'TO ZERO']
```

Comment:
statuuuuuuuuuus :)

Reply:
It's plural in Latin, you uncultured peasant.

Reply:
sorry that you have to work with such ignorant people ;)